R/csv.R

```diff
@@ -136,33 +138,96 @@ pl$scan_csv = function(
   }
 
   ## call low level function with args
-  check_no_missing_args(rlazy_csv_reader, args)
-  unwrap(do.call(rlazy_csv_reader, args))
+  check_no_missing_args(new_from_csv, args)
+  unwrap(do.call(new_from_csv, args))
 }
 
-#' Shortcut for pl$scan_csv()$collect()
+#' New DataFrame from CSV
+#'
+#' @description
+#' Read a file from path into a polars DataFrame.
+#' @name read_csv
 #' @rdname IO_read_csv
+#' @param path Path to a file or URL. It is possible to provide multiple paths
```

Comment:
Is it possible to avoid writing the same description twice?
I tried using the `inheritParams` tag but it doesn't seem to work.

Why not avoid duplication by combining them into the same Rd file?

Reply:
>  I tried using the inheritParams tag but it doesn't seem to work.

Yeah, I don't know why, that's annoying

> Why not avoid duplication by combining them into the same Rd file?

I'd rather have separate files, but I think I saw somewhere that we could store the roxygen docs as "template" and re-use them, so I'll try to find this again

Reply:
We can include Rmd files in roxygen docs but not in the parameters section, so I don't have a solution for this. I'd still like to keep separate docs so that `scan_csv` and `read_csv` appears separately in the sidebar of "Reference" on the website